tests/integration/devfile/utils/utils.go

```diff
@@ -308,14 +308,14 @@ func ExecPushWithNewFileAndDir(projectDirPath, cmpName, namespace, newFilePath,
 	helper.CmdShouldPass("odo", args...)
 }
 
-// ExecWithRestartAttribute executes odo push with a command attribute restart
-func ExecWithRestartAttribute(projectDirPath, cmpName, namespace string) {
+// ExecWithHotReload executes odo push with hot reload true
+func ExecWithHotReload(projectDirPath, cmpName, namespace string) {
 	args := []string{"create", "nodejs", cmpName}
 	args = useProjectIfAvailable(args, namespace)
 	helper.CmdShouldPass("odo", args...)
 
 	helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), projectDirPath)
-	helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-with-restart.yaml"), filepath.Join(projectDirPath, "devfile.yaml"))
+	helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-with-hotReload.yaml"), filepath.Join(projectDirPath, "devfile.yaml"))
```

Comment:
can we improve this test by checking in the logs? I mean checking in the logs for the absence of a kill string like
```
time="2020-08-20T11:14:05Z" level=error msg="program stopped with error:signal: killed" program=devrun 
time="2020-08-20T11:14:05Z" level=info msg="Stopped by user, don't start it again" program=devrun 
```

Reply:
Done.